tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -2,33 +2,44 @@ ARG OS_VERSION=18.04
 FROM ubuntu:${OS_VERSION}
 
 ARG PYTHON_VERSION=3.5
-ARG OPENVINO_VERSION=2020.4
+ARG OPENVINO_VERSION=2021.1
 
 ADD scripts /tmp/scripts
 RUN /tmp/scripts/install_ubuntu.sh -p $PYTHON_VERSION -d EdgeDevice && \
     /tmp/scripts/install_deps.sh -p $PYTHON_VERSION -d EdgeDevice
 
 RUN apt update && apt install -y libnuma1 ocl-icd-libopencl1 && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN /tmp/scripts/install_openvino.sh -o ${OPENVINO_VERSION} && \
```

Comment:
If we are not using install_openvino.sh, can we remove it or are we still using it for something else